ynput/OpenPype
# Nuke - Allow ExtractReviewDataMov with no timecode knob

## Changelog Description
ExtractReviewDataMov allows to specify file type. Trying to write some other extension than mov fails on generate_mov assuming that mov64_write_timecode knob exists.

## Testing notes:
1. Publish render from nuke, using ExtractReviewDataMov publish plugin
2. Set the ExtractReviewDataMov extension to mxf (project_settings/nuke/publish/ExtractReviewDataMov/outputs/baking/extension) and publish again

With this PR, second step should not fail
